.gitignore

```diff
@@ -24,3 +24,5 @@ npm-debug.log
 /test/results
 # Generated by BrowserStack's Tunnelling application
 /local.log
+coverage/
+.nyc_output/
```

Comment:
These are ignored at any point in the project tree.  Did you want to ignore them only at the root?